php documentation updates (#2654)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/php/logging.md

$eventLogger->logEvent('foo', $record);
```

## Integrations for 3rd-party logging libraries

### Monolog

> We provide a
> [monolog handler](https://packagist.org/packages/open-telemetry/opentelemetry-logger-monolog)
> which can be used to send monolog logs to an OpenTelemetry-capable receiver:

Reviewer: ```suggestion
You can use the
[monolog handler](https://packagist.org/packages/open-telemetry/opentelemetry-logger-monolog) to send monolog logs to an OpenTelemetry-capable receiver:
```
